### PR TITLE
User configuration tab datatable

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/app/assets/javascripts/webui2/users_groups.js [hunk 1]
```diff
@@ -1,48 +1,61 @@
+function initializeUserConfigurationDatatable() { // jshint ignore:line
+  projectsDatatable(
```
Comment (dmarcoux) [inline src/api/app/assets/javascripts/webui2/users_groups.js]: `projectsDatatable` seems to be generic enough to be reused, so we could rename it to `initializeServerSideDatatable` and move it under [app/assets/javascripts/webui2/datatables.js](https://github.com/openSUSE/open-build-service/blob/ed035576b5e8421c6a4583b9ea919601d36489ac/src/api/app/assets/javascripts/webui2/datatables.js)

Comment (bgeuken) [reply src/api/app/assets/javascripts/webui2/users_groups.js]: True that. I'll change it.

Comment (bgeuken) [reply src/api/app/assets/javascripts/webui2/users_groups.js]: @dmarcoux updated please have another look

File: src/api/app/helpers/webui/user_helper.rb [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
+    safe_join(
+      [
+        link_to(user_edit_path(user.login)) do
+          content_tag(:i, nil, class: 'fas fa-edit text-secondary pr-1', title: 'Edit User')
+        end,
+        mail_to(user.email) do
+          content_tag(:i, nil, class: 'fas fa-envelope text-secondary pr-1', title: 'Send Email to User')
+        end,
+        link_to(user_delete_path(user: { login: user.login }), method: :delete, data: { confirm: 'Are you sure?' }) do
+          content_tag(:i, nil, class: 'fas fa-times-circle text-danger pr-1', title: 'Delete user')
```
Comment (dmarcoux) [inline src/api/app/helpers/webui/user_helper.rb]: title: 'Delete **U**ser'

File: src/api/app/views/webui/user/index.html.haml [hunk 3]
```diff
@@ -13,14 +13,14 @@
     %table.compact#user-table
       %thead
         %tr
-          %td
+          %th
             User
           - if Configuration.ldap_enabled?
-            %td
+            %th
               Local user
```
Comment (dmarcoux) [inline src/api/app/views/webui/user/index.html.haml]: Local **U**ser

File: src/api/app/views/webui2/webui/user/index.html.haml [hunk 4]
```diff
@@ -4,42 +4,23 @@
...
   .card-body
     %h3= @pagetitle
-    %table.responsive.table.table-sm.table-striped.table-bordered#user-table
+    %table.responsive.table.table-sm.table-striped.table-bordered#user-table{ data: { source: users_path } }
       %thead
         %tr
-          %td
+          %th
             User
-          - if Configuration.ldap_enabled?
```
Comment (dmarcoux) [inline src/api/app/views/webui2/webui/user/index.html.haml]: Should this `if` be removed? It seems like it should not.

Comment (bgeuken) [reply src/api/app/views/webui2/webui/user/index.html.haml]: It should. The visibility of this column is now set during the [datatable initialization](https://github.com/openSUSE/open-build-service/pull/7108/files#diff-c76d56f625bcb8033c7090e2efe29d64R8)

**Comment by bgeuken** [comment]
@dmarcoux Updated. Please have another look

File: src/api/app/views/webui2/webui/user/index.html.haml (hunk 4 above)
Comment (dmarcoux) [reply src/api/app/views/webui2/webui/user/index.html.haml]: Oh okay, nice!

**Comment by dmarcoux** [comment]
Great job!

**Pull request merged**